### ISSUE TITLE
Datepicker highlights the selected day of the month across all months

### ISSUE THREAD

**Issue by alisagez** [salesforce/design-system-react#1402]
When a date is selected on the datepicker, that same date gets highlighted across all of the months.
Here's a video: https://www.screencast.com/t/SrHCLzoS8Er

**Comment by interactivellama** [comment]
That is focus and not selection. Where do you think keyboard user focus should be set when the month changes via the mouse? Should it be on the right arrow button when it is clicked?

Personally, I like being able to move the focus with the keyboard while clicking through the months, but I'm open to moving the focus to the month arrow button buttons when they are clicked.

What are you expecting to happen?
